### ISSUE TITLE
Unhealty container and can't login into. Just web page stays at the login page.

### ISSUE THREAD

**Issue by zaywalker** [jc21/nginx-proxy-manager#726]
**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
- Are you sure you're not using someone else's docker image?
- If having problems with Lets Encrypt, have you made absolutely sure your site is accessible from outside of your network?

**Describe the bug**
- A clear and concise description of what the bug is.

```
I haven't checked when this unhealty container starts. the portainer shows me I got unheathy npm container.
I've been using npm for a long time with watchtower auto docker updater and there was no problem.

The symtom is

1. portainer show me unhealty tag
2. container health output is "parse error: Invalid numeric literal at line 1, column 7 NOT OK"
3. All of my web site can access with the address which are created with npm
4. npm login page not working properly. I can't login
```
- What version of Nginx Proxy Manager is reported on the login page?
```
v 2.7.0
latest tag docker image.
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Operating System**
- Please specify if using a Rpi, Mac, orchestration tool or any other setups that might affect the reproduction of this error.
```
Linux 5.4.0-0.bpo.4-amd64 #1 SMP Debian 5.4.19-1~bpo10+1 (2020-03-09) x86_64 GNU/Linux
```

**Additional context**
Add any other context about the problem here, docker version, browser version if applicable to the problem. Too much info is better than too little.

**Comment by Havoc85** [comment]
Same here. And Probably same as: https://github.com/jc21/nginx-proxy-manager/issues/727

**Comment by zaywalker** [comment]
It seems to fix with 2.7.1. thanks!
